vector/src/main/java/im/vector/app/core/resources/UserPreferencesProvider.kt

```diff
@@ -52,4 +52,8 @@ class UserPreferencesProvider @Inject constructor(private val vectorPreferences:
     fun areThreadMessagesEnabled(): Boolean {
         return vectorPreferences.areThreadMessagesEnabled()
     }
+
+    fun isLiveSenderInfo(): Boolean {
```

Comment:
rename to `useLiveSenderInfo` ? Applicable to the other instances of this wording

Reply:
Done. Thank you.